src/atc_tools/helpers/module_helper.py

```diff
@@ -0,0 +1,74 @@
+import importlib
+import pkgutil
+import sys
+from inspect import getmembers, isclass
+from types import ModuleType
+from typing import Dict, Union
+
+
+class ModuleHelper:
+    @staticmethod
+    def get_modules(package: Union[str, ModuleType]) -> Dict[str, object]:
```

Comment:
If the return dict contains all modules - shoulnt the return type be: Dict[str,ModuleType]?